Add ngnix cluster resource

mpagot opened the pull request:
Add cluster resource to monitor the ngnix service, add it to the group with all the others.
Add a dedicated test to target the service directly.

lpalovsky commented on azure/az_lb_http_pacemaker/azure_lb_cluster_test_01_crm_mv.sh:
```diff
@@ -67,13 +72,16 @@ test_step "Check ip address immediately after the move"
 sleep 30
 test_web "${MYNAME}-vm-02"
 
+# Repeate the same but this time from VM-02 to VM-01
```
s/Repeate/Repeat/

@coderabbitai Take this you looser AI. I have found a typo you have missed!

lpalovsky replied:
@coderabbitai Stop pretending to be nice. You cannot fool me, AI. I know what are you planning!

lpalovsky reviewed (approved):
LGTM

alvarocarvajald reviewed (approved):
LGTM

mpagot committed:
Add ngnix cluster resource

Add cluster resource to monitor the ngnix service, add it to the group
with all the others.
Add a dedicated test to target nginx directly.
